sections/futures/UserInfo/UserInfo.tsx

```diff
@@ -105,7 +105,6 @@ const UserInfo: React.FC<UserInfoProps> = ({ marketAsset }) => {
 			<TabPanel name={FuturesTab.POSITION} activeTab={activeTab}>
 				<PositionCard
 					position={futuresMarketsPosition ?? null}
-					positionHistory={positionHistory ?? null}
```

Comment:
The position history is blank now, I'm assuming it was this change. Is there a reason to remove this?

Reply:
The position history data I was pulling was incorrect. It was only used to calculate average entry price, which we already had from the subgraph (I think).

Reply:
Ok, we will get this from the subgraph long-term. Can you disable this tab/button?